app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
-class ApplicationController < ActionController::Base
+class ApplicationController < ActionController::API
+  include ActionController::RequestForgeryProtection
```

Comment:
I think we can ditch the request forgery protection altogether (including the method call below). I don't think it'll help with our use case.